ui/src/frontend/css_constants.ts

```diff
@@ -41,14 +37,12 @@ export function initCssConstants() {
       OVERVIEW_TIMELINE_NON_VISIBLE_COLOR;
   DEFAULT_DETAILS_CONTENT_HEIGHT =
       getCssNum('--details-content-height') || DEFAULT_DETAILS_CONTENT_HEIGHT;
-  BACKGROUND_COLOR = getCssStr('--main-background-color') || BACKGROUND_COLOR;
-  FOREGROUND_COLOR = getCssStr('--main-foreground-color') || FOREGROUND_COLOR;
 }
 
-function getCssStr(prop: string): string|undefined {
+export function getCssStr(prop: string): string|undefined {
```

Comment:
Every new call site uses a bang to assert a defined return result. Should the `getCssStr` function perhaps have a second argument for a default value in case the variable doesn't resolve? Alternatively, there are many extant call sites that provide their own defaults via `||`. Probably we should do one or the other and not the bang assertion.

Does @martin-fleck-at have an opinion?

Reply:
Yeah, I actually agree that a default value might be a good non-breaking change for this function that we can probably get accepted as well. But I'm also fine with the way it is now if we properly ensure that undefined is handled.